test/runtests.jl

```diff
@@ -1,20 +1,22 @@
 using LinearAlgebra
 using SparseArrays
 using Test
+using Random
 
 using AMD
 import AMD: AMD_STATUS, AMD_OK, COLAMD_STATUS, COLAMD_OK
 
-_Clong = Base.Sys.WORD_SIZE == 32 ? Clong : Clonglong
+Random.seed!(666)
+
 for n in [10, 20, 30]
   for density in [0.25, 0.75, 1.0]
-    for T in [Cint, _Clong]
+    for T in [Cint, Int64]
```

Comment:
Is `_Clong` always guaranteed to be an `Int64`?

Reply:
_Clong was implemented as
```julia
_Clong = Base.Sys.WORD_SIZE == 32 ? Clong : Clonglong
```
which is equivalent to `_Clong = Int64` for all platforms.

Reply:
I'm not so sure. Clong is system dependent. It could be an Int on 32-bit systems: https://docs.julialang.org/en/v1/manual/calling-c-and-fortran-code/#mapping-c-types-to-julia

Reply:
SuiteSparse 5.x.x is used by Julia >= 1.6 and <1.10 and its SuiteSparse_long is a 64 bits integer for all 64 bits platforms. `_Clong` represents this `SuiteSparse_long`.

They have a check in this [header file](  https://github.com/DrTimothyAldenDavis/SuiteSparse/blob/v5.13.0/SuiteSparse_config/SuiteSparse_config.h#L54) to verify if the the platform is a 64 bits Windows or not.

For 32 bits platforms, SuiteSparse_long is indeed Clong / Int32. We dropped support for these platforms in JSO but let's not break that if we can.